deploy/rbd/kubernetes/csi-snapshotter.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: csi-snapshotter
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: csi-snapshotter
+  labels:
+    app: csi-snapshotter
+spec:
+  selector:
+    app: csi-snapshotter
+  ports:
+    - name: dummy
+      port: 12345
+---
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: csi-snapshotter
```

Comment:
name: csi-rbd-snapshotter (we'll have cephfs snapshotter later)

Reply:
as per suggestion moved into provisioner (so no need for this one)